.github/workflows/main.yml

```diff
@@ -56,38 +56,6 @@ jobs:
         run: mkdir -p build && cd build && cmake -DCMAKE_BUILD_TYPE=Debug .. && make
       - run: make cpp_test
 
-  build_ledger:
```

Comment:
I think we can leave the build_ledger CI to make sure we are correctly compiling the app using nanoX, remove the size_nano_s and trigger the other actions if this step was successful